### ISSUE TITLE
Cart not updated after deleting product

### ISSUE THREAD

**Issue by templatin** [PrestaShop/ps_shoppingcart#53]
This issue is seen after upgrading the module to v2.0.3. Cart in header is not updated after deleting the product. Its updated only after page reload.

**To reproduce**
1. Add any product to cart.
2. Go to cart page and delete the product.
3. Cart in header is not updated.

**Additional information**
Prestashop version: 1.7.6.2
Module version: 2.0.3

**Screenshot**
![cart](https://user-images.githubusercontent.com/3615808/70118884-1a8ffd80-168f-11ea-9b09-fcbc57fc9ca7.jpg)

**Comment by khouloudbelguith** [comment]
Duplicate of https://github.com/PrestaShop/PrestaShop/issues/16665
